pkg/base/linuxutils/lsblk/lsblk.go

```diff
@@ -161,11 +161,6 @@ func (l *LSBLK) GetBlockDevices(device string) ([]BlockDevice, error) {
 // Receives an instance of drivecrd.Drive struct
```

Comment:
one general comments is to create an issue first and link the issue to the PR.  and put [ISSUE-xxx] in PR title.

Reply:
Addressed